README.md

```diff
@@ -299,8 +301,8 @@ class fourthcoffee(
 }
 ~~~
 
-As you can see, you can mix and match dsc resources with common puppet resources.
-All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metaparameter.html) should also be supported.
+As you can see, you can mix and match dsc resources with common Puppet resources.
```

Comment:
The name dsc, when it refers to the module, should be lowercased; this isn't that, however. Good catch!